### ISSUE TITLE
September Milestone Task #5: Standardize Data

### ISSUE THREAD

**Issue by sbalagali** [Break-Through-Tech/MathWorks-1D-unlocking-embedded-ai-through-efficient-compression#4]
Standardize/fit the training set. Then transform the test set using the same transformation from the training set. Save standardized data for model training.
